### ISSUE TITLE
Export poetry dependencies without `python_version` interval

### ISSUE THREAD

**Issue by brunolnetto** [python-poetry/poetry#7755]
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [FAQ](https://python-poetry.org/docs/faq/) and general [documentation](https://python-poetry.org/docs/) and believe that my question is not already covered.

## Feature Request

I have been looking for a way to export my project dependencies. I know there is the command `poetry export` with its flags, which performs what I am looking for. However, I added additionally the flags ` --without-hashes` and ` --without-urls` to turn it less verbose than before.

On the conventional exporting way with `pip freeze`, the format corresponds to `package_name==package_version`. I can perform this same pattern on Linux with the command below,  but it would be great to have an additional flag `--without-pyversion`

`poetry export --without-hashes --without-urls | awk '{ print $1 }' FS=';'`

**Comment by dimbleby** [comment]
this issue belongs in https://github.com/python-poetry/poetry-plugin-export/, not here

however I expect the answer should be "no": poetry makes you declare the range of python versions that you support and the export faithfully reflects that.

Also simply removing the python version markers indiscriminately is definitely wrong: what if some project uses `foo==1.0.0` for python versions 3.7 and 3.8, but `foo==2.0` for python versions 3.9 and higher?

**Comment by brunolnetto** [comment]
I do not think it is "black" on this right-wrong categorization, but certainly, there is this pathway in which pip resolves dependencies for itself.

However, I do agree with your remark that poetry brings fresh air akin to this gray-and-fuzzy feature in package management.
